src/components/DataTableRow/DataTableRow.js

```diff
@@ -0,0 +1,20 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const DataTableRow = props => {
+  const { className, children, ...other } = props;
+
+  return (
+    <tr {...other} className={className}>
+      {children}
+    </tr>
+  );
+};
+
+DataTableRow.propTypes = {
+  children: PropTypes.node,
+  className: PropTypes.string,
+  containerClassName: PropTypes.string,
```

Comment:
Seems like this is an extra prop, potentially, when `classnames` was being used?

Reply:
Yup, removing it! 😄